metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -17,6 +17,7 @@
 import pydantic
 import requests
 from expandvars import expandvars
+from requests_file import FileAdapter
```

Comment:
Note that intent here is to exercise the same logic that fetches compose files from github to be able to use file:// in the same way. There is logic in this class that is generally useful for running the smoke test locally.